WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+
+The flying ship has an Armor Class of 15, and 300 hit points; immunity to poison and psychic damage, and immunity to all conditions.
+
+<div class='footnote'>PART 2 | SPELLS </div>
+
+\pagebreakNum
+
+It has a maximum carrying capacity of 50 tons (limited by the size of the ship itself). It is not slowed by any weight until its maximum capacity is reached, at which point the ship loses its flying property and falls at a rate of 60 feet per round until it hits the ground. Taking no falling damage from the fall, and leaving its passengers unharmed.
+
+You can this effect permanent by casting this spell on the same ship every day for one year.
+
+#### Living Bomb
+*2nd-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 60 feet
+- **Components:** V, S, M (a flammable liquid)
+- **Duration:**  Concentration, up to 1 minute
+___
+You manipulate the mana flowing through a creature within range, making it burn from the inside with a roaring fire. The target must make a Constitution saving throw. Taking 1d10 fire damage on a failed save, or half as much on a successful one. At the start of each of your turns for the spells duration, the target must make another Constitution saving throw as it takes the damage again.
+
+When the spell ends the target and each creature within 10 feet of it must succeed on a Dexterity saving throw or take 2d6 fire damage.
+
+***At Higher Levels.*** When you cast this spell using a spell slot of 3rd level or higher, the damage of the explosion increases by 1d6 for each slot level above 2nd.
+
+#### Pyroblast
+*7th-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S, M (a piece of brimstone)
+- **Duration:**  Instantaneous
+___
+You clasp your hands together, conjuring a ball of dripping magma, launching it at a creature or object within range. Make a ranged spell attack against the target. On a hit, the target takes 13d10 fire damage and ignites. Until a creature takes an action to douse the fire, the target takes 1d10 fire damage at the start of each of its turns.
+
+***At Higher Levels.*** When you cast this spell using a spell slot of 8th level or higher, the initial damage increases by 1d10 for each slot level above 7th.
+
+#### Spellsteal
+*3rd-level abjuration*
+___
+- **Casting Time:** 1 action
+- **Range:** 60 feet
+- **Components:** V, S
+- **Duration:** Concentration, up to 1 minute
+___
+Choose one creature within range. Any spell of 3rd level or lower that affects the target is transferred to you for the duration of spellsteal. If the spell requires concentration, it must still be maintained by its caster. Concentration must be broken by both its original caster and you before the spell and its effects disappear. To steal a spell of 4th level or higher, make an ability check using your spell&shy;casting ability. The DC equals 10 + the spell's level. On a successful check, you steal the effects of the spell.
```

Comment:
Is it intentional that, if the spell was concentrated, all the original caster has to do in order to break the spell for the mage ... is to just stop concentrating? If so, that'd make spellsteal a bit of a challenge in most scenarios where the mage would be using it, right?

Maybe it just reads like it's a strange contradiction. Because concentration must be broken by both parties for the spell to end, at the same time as it must still be maintained by its caster. Must it still be maintained by its caster in order to work, or is its caster forced to still concentrate the spell for the duration of the spellsteal?

I might be wrong, but I think there's also an implication here that an "until the end of your next turn" type spell effect could stretch out to 1 minute. Not sure how many of those there are though, the majority case for spellsteal would be concentration spells.

Reply:
Choose one creature within range. A spell, of your choosing, of 3rd level or lower that affects the target is transferred to you for the duration of spellsteal. If the spell requires concentration and the original caster keeps concentrating on it, it is transferred back to its original target when spellsteal ends an. To steal a spell of 4th level or higher, make an ability check using your spell&shy;casting ability. The DC equals 10 + the spell's level. On a successful check, you steal the effects of the spell.